### DISCUSSION TITLE
I cant acces bootstrap docs!

### DISCUSSION THREAD

**Discussion by AbderrazagB** [twbs/bootstrap#39841] [Q&A]
Please help me, getbootstrap.com was working fine until today it stopped and I cant access the website anymore. 
I don't understand what's the problem

**Comment by julien-deramond** [comment]
Hey there!

> until today it stopped and I cant access the website anymore

What's the error message in the browser? Screenshot? Errors in the console?

**Reply by AbderrazagB** [reply]
![image](https://github.com/twbs/bootstrap/assets/113622795/8196da0d-7aea-4462-b764-e405008e3f89)

This is what I m getting, it seems like a problem with my internet provider because when I try a different network it works fine.

**Reply by initrohit** [reply]
Brother, There is a problem with your network provider not with the boostrap.
